bqn-mode.el

```diff
@@ -1,14 +1,23 @@
-;;; bqn-mode --- Emacs mode for BQN -*- lexical-binding: t -*-
-;;;
+;;; bqn-mode.el --- Emacs mode for BQN -*- lexical-binding: t -*-
+
+;; BQN's emacs mode is derived from gnu-apl-mode,
+;; which is copyright 2013-2015 Elias Mårtenson <lokedhs@gmail.com>.
+;; Changes are copyright 2021 Marshall Lochbaum <mwlochbaum@gmail.com>.
+
+;; Author: Marshall Lochbaum <mwlochbaum@gmail.com>
+;; Version: 0.1.0
```

Comment:
As we need a version, put the smallest one:

```suggestion
;; Version: 0.0.0
```

Reply:
> As we need a version, put the smallest one:

Looks like the ["official" style guide](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase) recommends `0.1.0` (which I wasn't aware of at the time of choosing it), though I completely disagree with their characterization of it being the simplest...

Reply:
I want to reserve `0.1.0` for a somewhat complete release.
More accurately, I am studying the patch #7, and merged some changes. When I finished with it, I will tag/release the 0.1.0.